samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,186 @@
+    - `OnlineMeetings.Read.All`
+    - `OnlineMeetings.ReadWrite.All`
+    - `OnlineMeetingTranscript.Read.All`
+    - `User.Read.All`
+    - `User.ReadBasic.All`
+    - `User.ReadWrite.All`
+
+        ![Application Permission](Images/ApplicationPermission.png)
+
+1) Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+
+## Try it yourself - experience the App in your Microsoft Teams client
+Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
+
+**Microsoft Teams Meeting Helper sample app:** [Manifest](/samples/bot-ai-meeting-helper/demo-manifest/bot-ai-meeting-helper.zip)
+
+- To run the demo, you must set up and grant the necessary policy. Please follow the second step mentioned above for instructions on how to do this. 
+
+## Running the sample
+
+1) **Select Upload an app and choose the app package if you are running demo-manifest then choose downloaded demo-manifest zip file. If the sample is running locally then root folder, appPackage >> build >> appPackage.local.zip**
+![UploadCustomeApp](Images/0.UploadCustomeApp.png)
+
+1) **To fetch all upcoming meetings when a user requests "What is my upcoming meetings," the application will display all future scheduled one-time meetings and recurring meetings, showing individual instances of each recurring meeting.**
+![MeetingHelperSearchApp](Images/2.MeetingHelperFindUpcomingMeeting.png)
+
+1) **When the user clicks on a specific meeting, the application will call the Graph API to create a subscription for that particular instance, store the subscription details in the database, and then send an acknowledgment.**
+![MeetingHelperSelectApp](Images/3.MeetingHelperSubscription.png)
+
+1) **Once the meeting has started, the user needs to join the meeting.**
+![MeetingHelperSearchFileName](Images/4.MeetingHelperJoinMeeting.png)
+
+1) **After joining the meeting, the user must navigate to the "More options" menu at the top and select "Record and transcribe," then choose "Start transcription" to begin recording the transcription.**
+![MeetingHelperSelectResults](Images/5.MeetingHelperStartTranscription.png)
+
+1) **Once the meeting is concluded, the transcription will be generated in the background. Additionally, the application will internally call the Graph API to obtain the transcription. Using Azure OpenAI, it will extract all meeting action items and generate a meeting summary.**
+![6.MeetingHelperResults](Images/6.MeetingHelperEndMeeting.png)
+
+1) **All subscribed users will automatically receive the meeting summary and action items.**
+![7.MeetingHelperYesResults](Images/7.MeetingHelperResults.png)
```

Comment:
Add deployment step as well .
Refer this PR: https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1335/files#diff-820624b3d7f7e4be48d0fc48df9ab543ee07089829a4bf35bd2e808c12637bb9R138

Reply:
Done